### ISSUE TITLE
Comparison to git-crypt

### ISSUE THREAD

**Issue by t3hmrman** [share-secrets-safely/cli#15]
It would be nice if the documentation would how `sheesy` differentiates itself from a `git-crypt`.

So far as I can see currently:

- No integration with `git`
- No support for simple AES key encryption

Maybe it doesn't make sense to compare `sheesy` to a tool like `git-crypt` in the first place, since the scopes are somewhat different, if so I apologize! I think that sheezy could be super useful for storing secrets right in git repos so that's one way I really want to use it.

P.S. I just saw the talk @ RustCologne -- cool project!

**Comment by Byron** [comment]
Indeed, and I agree that it would be nice to have. The reason it's not there yet is that I have never even looked at the tool.
From what I see, the bullet points provided here seem accurate, maybe there is more.

With the hopes of you getting more familiar with `sheesy`, maybe you would be able to deliver a comparison at some point.

**Comment by t3hmrman** [comment]
I would love to contribute documentation on it! As soon as I find a free weekend I'll send a PR! :)
